CMakeLists.txt

```diff
@@ -35,10 +35,10 @@ if(NBL_STATIC_BUILD)
 	message(STATUS "Static Nabla build enabled!")
 else()
 	if(MSVC)
```

Comment:
remove the `MSVC` check here - for `NOT NBL_STATIC_BUILD` case we should only check now if `NBL_COMPILER_DYNAMIC_RUNTIME` is enabled otherwise throw an error to make sure we have dynamic runtime for dynamic builds